examples/facebook/posts/posts.bal

```diff
@@ -52,9 +52,10 @@ service /facebook on fbListener {
     resource function get posts() returns string[]|error {
```

Comment:
Lets mark these resources as isolated. Better check in other places as well.

Reply:
Fixed in 3fa5fad and 92b35d4